lib/backburner/utils.ts

```diff
@@ -3,7 +3,7 @@ const NUMBER = /\d+/;
 export const now = Date.now;
 
 export function each<T>(collection: T[], callback: (v: T) => void) {
-  for (let i = 0; i < collection.length; i++) {
+  for (let i = 0, l = collection.length; i < l; i++) {
     callback(collection[i]);
```

Comment:
here is a fun read relating to this: http://mrale.ph/blog/2014/12/24/array-length-caching.html